Projects/hCoreUI/Sources/hForm/hFloatingTextField.swift

```diff
@@ -311,37 +311,27 @@ struct hFloatingTextField_Previews: PreviewProvider {
     }
 }
 
-private struct EnvironmentHWithoutDisabledColor: EnvironmentKey {
-    static let defaultValue = false
+public enum BackgroundColorType {
```

Comment:
Really good idea to handle them like this, but naming maybe needs to be updated like
`BackgroundOption` or something similar

Reply:
Yeah that sounds good! Updated it 👍